Client/src/pluginConfig.ts

```diff
@@ -3,11 +3,11 @@ import {
   WordCloudAnalysisPlugin,
   StepAnalysisViewPlugin,
   ResultTableSummaryViewPlugin,
-  StepAnalysisDefaultForm,
-  StepAnalysisDefaultResult,
-  StepAnalysisEupathExternalResult,
 } from '@veupathdb/wdk-client/lib/Plugins';
 import { ClientPluginRegistryEntry } from '@veupathdb/wdk-client/lib/Utils/ClientPlugin';
+import { StepAnalysisDefaultForm } from './plugins/StepAnalysisDefaultForm';
+import { StepAnalysisDefaultResult } from './plugins/StepAnalysisDefaultResult';
```

Comment:
Once you've put `StepAnalysisDefaultForm` and `StepAnalysisDefaultResult` back into `WDKClient`, you'll have to update these imports, of course.

Reply:
I've reverted the location of these `StepAnalysisDefault` components both here and in `WDKClient`.